masters/views.py

```diff
@@ -1,3 +1,41 @@
-from django.shortcuts import render
+import json, bcrypt
 
-# Create your views here.
+from django.views import View
+from django.http import JsonResponse
+
+from masters.models import Master
+
+class MasterSignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+            
+            if not Master.validate(data):
+                return JsonResponse({"message":"VALIDATION_ERROR"}, status=401) 
+
+            name            = data["name"]
+            password        = data["password"]
+            phone           = data["phone"]
+            gender          = data["gender"]
+            email           = data["email"]
+            birth           = data["birth"]
+            region          = data["region"]
+            main_service    = data["main_service"]
+            career          = data["career"]
+            hashed_password = bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt())
+
+            Master.objects.create(
+                name         = name,
+                password     = hashed_password.decode(),
+                phone        = phone,
+                gender       = gender,
+                email        = email,
+                birth        = birth,
+                region       = region,
+                main_service = main_service,
+                career       = career,
+            )
```

Comment:
```suggestion
            password        = data["password"]
            hashed_password = bcrypt.hashpw(password.encode("utf-8"), bcrypt.gensalt())

            Master.objects.create(
                name         = data['name'],
                password     = hashed_password.decode(),
                phone        = data['phone'],
                gender       = data['gender'],
                email        = data['email'],
                birth        = data['birth'],
                region       = data['region'],
                main_service = data['main_service'],
                career       = data['career'],
            )
```
여러번 사용되는 값들이 아니니까 다음과 같이 바꾸는 것이 가독성이 더 좋을 것 같습니다.

Reply:
수정 했습니다!